test/http/http.test.js

```diff
@@ -62,6 +75,12 @@ test('http can accept a single object argument', () => {
   })
 })
 
+test('http throws an error if `url` is not provided', async () => {
+  await expect(http()).rejects.toThrow()
+  await expect(http("", {})).rejects.toThrow()
+  await expect(http({ url : null })).rejects.toThrow()
```

Comment:
Tests the new error message and gets us test coverage for line 83 (defaulting to the empty object)

Reply:
When we have multiple expectations, is it essentially an `AND` for passing? And if it fails on 1 expectation, does it show "2 of 3" pass for example in terminal?

Reply:
Yes, they're treated as an AND. If the code is modified to only throw on `""`, then the test will fail.

However, they aren't collected. So in the example above, the rest runner will flag line 79 as failing (returns a resolved promise instead of throwing), but it will not flag 81 for failing.

Are you ok with that behavior?

Reply:
Yup that's fine for me (mostly just confirming my understanding)